src/common/mavlink.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#pragma GCC diagnostic push
+/*
+ * All structs in mavlink are packed since they are "on-the-wire" definition,
+ * but then pymavlink generates helper functions that take the address of
+ * members of those structs - that's not good, but there's nothing we can do,
+ * so shut up the warnings.
+ *
+ * Unaligned access is generally slower, but should work in most of today
+ * platforms, so this is not harmful. If you have a, e.g. sparc, mips, arm or
+ * ... without unaligned access, you may be interested in fixing
```

Comment:
None of the ARM CPUs we use in drones hardfault on unaligned access.

Reply:
pymavlink can still be fixed. In the cases it's not harmful, it could use macros to store/retrieve the values. This needs to be checked case by case.

Depending on what's the use for unaligned access, something like what I use in kmod could make it: https://git.kernel.org/pub/scm/utils/kernel/kmod/kmod.git/tree/shared/util.h#n49

Reply:
No doubt this needs to be addressed in pymavlink.
My comment was that there will not actually be an unaligned access, it is just a compiler warning, and thus it will still work on all these platforms.

Reply:
I reworded the comment to handle what you said.